src/app/lib/analyticsUtils/index.js

```diff
@@ -125,6 +125,19 @@ export const getReferrer = (platform, origin, previousPath) => {
   return null;
 };
 
+export const getAtUserId = platform => {
+  if (platform === 'amp') {
+    return null;
+  }
+
+  if (onClient()) {
+    const atuserid = Cookie.getJSON('atuserid');
```

Comment:
To anyone else curious, `Cookie.getJSON` is fail safe :D https://github.com/js-cookie/js-cookie/blob/155e16b95aa39afc9e8ddbc202a065afee5866b5/src/js.cookie.mjs#L96-L98